lib/services/local/session.go

```diff
@@ -89,6 +90,27 @@ func (s *IdentityService) DeleteAppSession(ctx context.Context, req types.Delete
 	return nil
 }
 
+// DeleteAllAppSessions removes all application web sessions.
+func (s *IdentityService) DeleteUserAppSessions(ctx context.Context, req *proto.DeleteUserAppSessionsRequest) error {
```

Comment:
Is this Identity method actually used anywhere? In the ACL layer we still delete each session individually AFAICS. Is it needed to just satisfy some interface?

Reply:
I've changed the implementation to use this function on the ACL layer.